app/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.hedvig.app">
+
+    <application android:extractNativeLibs="true" />
```

Comment:
Did this help decrease the .APK size?

Reply:
No - it is however required to use `MockK`. It only applies for builds with the `androidTest`-configuration, so it should not affect production builds at all.